src/Simulation/EntryPointDriver/Driver.cs

```diff
@@ -208,6 +214,41 @@ private void AddOptionIfAvailable<T>(Command command, OptionInfo<T> option)
                     $"The required option {option.Aliases.First()} conflicts with an entry point parameter name.");
             }
         }
+
+        /// <summary>
+        /// Adds a validator to the command such that the specified options cannot be simultaneously included.
+        /// </summary>
+        /// <param name="command">The command to add the validator to.</param>
+        /// <param name="primaryAliases">The primary aliases of the options to be marked as mutually exclusive.</param>
+        /// <remarks>Options must already be added to the command before invoking this method.</remarks>
+        private void MarkOptionsAsMutuallyExclusive(Command command, string[] primaryAliases) =>
+            command.AddValidator(result =>
+            {
+                var presentAliases = new List<string>();
+                foreach(var rawAlias in primaryAliases)
+                {
+                    var (prefix, alias) = Options.PrefixAliasTuple(rawAlias);
+                    var option = command.Options.Where(o => o.Aliases.Contains(alias)).FirstOrDefault();
```

Comment:
I think if instead of `o.Aliases`, you used `o.RawAliases` which includes the `--` or `-`, you could do this without needing `PrefixAliasTuple`?

Reply:
Possibly, I was not aware of the RawAliases property. I'll give it a try and as you mentioned maybe the PrefixAliasTuple is not needed anymore.

Reply:
Using `o.RawAliases` now.